content/en/synthetics/settings/_index.md

```diff
@@ -118,7 +118,18 @@ For more information about TOTP-based MFA in a browser test, see [TOTPs For Mult
 
 Once created, global variables can be used in all Synthetic tests. To import your global variables into your test, click **+ Variables**, type `{{` in a field you want to add the variable, and select your global variable. 
 
-For more information, see the [HTTP test][8], [Multistep API test][9], [Browser test configuration][10], and [Steps documentation][16].
+To update the value of an existing global variable using [Terraform][17], create a global variable with the same name, description, and tags, but different value. 
```

Comment:
I do not think we should talk about terraform here. It adds complexity and it is the only place in the all doc we have it. I would recommend we keep TF documentation on the TF repository